### DISCUSSION TITLE
Animated HEIC Files, part 2

### DISCUSSION THREAD

**Discussion by Jay123210599** [ImageMagick/ImageMagick#7755] [Help]
### ImageMagick version

7.1.1-27

### Operating system, version and so on

Windows 11

### Description

I have built ImageMagick from the source using Visual Studio. Now how do I recompile it with libheif?

**Comment by dlemstra** [comment]
You are doing a recompile when you build ImageMagick from source using Visual Studio. Not sure what you are asking for?

**Reply by Jay123210599** [reply]
I want ImageMagick to make heif/heic files, but it needs libheif to do it, and the pre-built exe installer versions don't have it.

**Reply by dlemstra** [reply]
The pre-built installer contains libheif but it doesn't contain the x265 encoder. And we haven't setup anything to make it possible to build the x265 library. This will be something that you will need to figure out yourself.

**Reply by Jay123210599** [reply]
maybe we can add the encoder when building it from source?

How about this?
https://github.com/ImageMagick/heif

**Reply by dlemstra** [reply]
This is something that you will need to figure out.
